packages/micropip/src/micropip/package.py

```diff
@@ -31,6 +32,10 @@ def format_row(values, widths, filler=""):
     return "\n".join(rows)
 
 
+def normalize_package_name(pkgname: str):
+    return re.sub(r"[^\w\d.]+", "_", pkgname, re.UNICODE)
```

Comment:
perhaps also apply `.lower()`?

Reply:
yes, indeed, that would definitely make sense. Thanks @bollwyvl !

Reply:
e.g. https://github.com/pyodide/pyodide/pull/2445